app/src/main/assets/shared_error_style.css

```
@@ -64,17 +64,31 @@ p {
 
 button {
   display: block;
-  height: 36px;
+  height: 48px;
   width: 100%;
-  border-radius: 5px;
   border: 0;
+  padding: 6px 0px;
   font-family: inherit;
-  background-color: var(--primary-button-color);
+  background-color: transparent;
   color: var(--primary-button-text-color);
   font-size: 14px;
   font-weight: bold;
-  margin: var(--moz-vertical-spacing) auto;
+  margin: var(--moz-vertical-spacing) 0;
   text-align: center;
+  position: relative;
+}
+
+button:after {
```

nit: psuedo elements usually use a double colon
```suggestion
button::after {
```